src/SuperGoodRobot/CLAW_ARM_TEST.ino

```diff
@@ -0,0 +1,24 @@
+void start_claw_arm_test() {
+	LCD.clear();
+	LCD.home();
+	setForearmServo(0);
+	setBasearmServo(0);
+}
+
+void claw_arm_test() {
+	int forearm_position=constrain((byte)map(knob(6),0,KNOB_MAX,0,180),0,180-1); //get the position by mapping the knob
```

Comment:
Fix commenting style here and everywhere. Comments should always have a space after the delimiter for legibility.
" // get the..." > good
" //get the..." > bad